### PR TITLE
Add channels, env and prettier

### PR DESCRIPTION
**Pull request by LouisLeNezet**
Issue #1389 

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1391"></a>

### CODE REVIEW

File: markdown/usage/tutorials/nf_core_usage_tutorial.md [hunk 1]
```diff
@@ -103,6 +103,15 @@ Once installed, you can check that everything is working by printing the help:
...
 
+You will also need to install [Prettier](https://prettier.io/) for formatting your code.
+To do so, you can either use the following command with conda:
+
+```bash
+conda install -c conda-forge prettier
+```
+
+Or use the Visual Studio Code extension [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) also available in the pack of useful extension [NF-core](https://marketplace.visualstudio.com/items?itemName=nf-core.nf-core-extensionpack).
+
```
Comment (mribeirodantas) [inline markdown/usage/tutorials/nf_core_usage_tutorial.md:114]:
```suggestion

Besides, you can also add a comment with @nf-core-bot fix linting in your Pull Request and prettier will be used to apply the required fixes to your code.

```

**Review by mribeirodantas — CHANGES_REQUESTED** [review]

**Comment by mribeirodantas** [comment]
It seems you accepted twice the suggestion 🤯. Please, remove the repeated sentence at the end.

**Comment by LouisLeNezet** [comment]
Yeah I saw that.
I'm trying to remove it but the master branch has changed...

File: markdown/usage/tutorials/nf_core_usage_tutorial.md [hunk 2]
```diff
@@ -103,6 +103,19 @@ Once installed, you can check that everything is working by printing the help:
...
+Besides, you can also add a comment with @nf-core-bot fix linting in your Pull Request and prettier will be used to apply the required fixes to your code.
```
Comment (mashehu) [inline markdown/usage/tutorials/nf_core_usage_tutorial.md:115]:
```suggestion
Besides, you can also add a comment with `@nf-core-bot fix linting` in your Pull Request and prettier will be used to apply the required fixes to your code.
```

File: markdown/usage/tutorials/nf_core_usage_tutorial.md [hunk 3]
```diff
@@ -103,6 +103,19 @@ Once installed, you can check that everything is working by printing the help:
...
+
+```bash
+conda install -c conda-forge prettier
+```
+
+Or use the Visual Studio Code extension [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) also available in the pack of useful extension [NF-core](https://marketplace.visualstudio.com/items?itemName=nf-core.nf-core-extensionpack).
+
+Besides, you can also add a comment with `@nf-core-bot fix linting` in your Pull Request and prettier will be used to apply the required fixes to your code.
+
+Besides, you can also add a comment with @nf-core-bot fix linting in your Pull Request and prettier will be used to apply the required fixes to your code.
```
Comment (mribeirodantas) [inline markdown/usage/tutorials/nf_core_usage_tutorial.md:117]:
```suggestion
```

**Review by mribeirodantas — APPROVED** [review]

File: markdown/usage/tutorials/nf_core_usage_tutorial.md [hunk 4]
```diff
@@ -103,6 +103,18 @@ Once installed, you can check that everything is working by printing the help:
...
+To do so, you can either use the following command with conda:
+
+```bash
+conda install -c conda-forge prettier
+```
+
+Or use the Visual Studio Code extension [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) also available in the pack of useful extension [NF-core](https://marketplace.visualstudio.com/items?itemName=nf-core.nf-core-extensionpack).
+
+Besides, you can also add a comment with `@nf-core-bot fix linting` in your Pull Request and prettier will be used to apply the required fixes to your code.
+
```
Comment (mribeirodantas) [inline markdown/usage/tutorials/nf_core_usage_tutorial.md:116]:
```suggestion
```

**Comment by LouisLeNezet** [comment]
Thanks for the help !

**Pull request merged**